modules/fedimint-ln-common/src/gateway_endpoint_constants.rs

```diff
@@ -0,0 +1,16 @@
+pub const ADDRESS_ENDPOINT: &str = "/address";
+pub const BACKUP_ENDPOINT: &str = "/backup";
+pub const BALANCE_ENDPOINT: &str = "/balance";
+pub const CONFIGURATION_ENDPOINT: &str = "/config";
+pub const CONNECT_FED_ENDPOINT: &str = "/connect_fed";
+pub const CREATE_INVOICE_V2_ENDPOINT: &str = "/create_invoice";
+pub const GATEWAY_INFO_ENDPOINT: &str = "/info";
+pub const GET_GATEWAY_ID_ENDPOINT: &str = "/id";
+pub const HANDLE_POST_INFO_ENDPOINT: &str = "/info";
```

Comment:
`HANDLE` is weird. Can this be named `GATEWAY_INFO_POST_ENDPOINT`?

Reply:
Done